sdk/lusid/utilities/api_client_factory.py

```diff
@@ -126,3 +142,11 @@ def init_impl(dest, src=None):
         setattr(metaclass, "__init__", init_impl)
 
         return api_impl
+
+    @property
+    def auth_method(self):
+        return self.__auth_method
```

Comment:
Added a new property to the `ApiFactory` class which helps users understand how the factor was authenticated.